sdk/src/fee.rs

```diff
@@ -33,9 +33,9 @@ pub struct FeeStructure {
 
 #[derive(Debug, Default, Clone, Copy, Eq, PartialEq, Deserialize, Serialize)]
 pub struct FeeDetails {
-    transaction_fee: u64,
-    prioritization_fee: u64,
-    remove_rounding_in_fee_calculation: bool,
+    pub transaction_fee: u64,
+    pub prioritization_fee: u64,
+    pub remove_rounding_in_fee_calculation: bool,
```

Comment:
Are the `pub`s necessary? They were intentionally kept private so `total_fee()` hides the messy internal details

Reply:
Alternatively I can make the `new` fn public - that seems better.

Reply:
e65eae5d43f8c44b3d18c57b919ce9204f4a3eb5